[INFO] Module GCC/9.4.0 loaded.

Inactive Modules:
  1) UCX/1.17.0     2) numactl/2.0.16

The following have been reloaded with a version change:
  1) GCCcore/.11.3.0 => GCCcore/.9.4.0     3) zlib/1.2.13 => zlib/1.2.11
  2) binutils/2.38 => binutils/2.36.1

[INFO] Module CUDA/11.8.0 loaded.
[INFO] Module cuDNN/8.6.0.163-CUDA-11.8.0 loaded.
Traceback (most recent call last):
  File "/home/md334167/elzerman-with-unet/train_for_exdata.py", line 326, in <module>
    main()
  File "/home/md334167/elzerman-with-unet/train_for_exdata.py", line 53, in main
    hdf5Data_noise.set_traces_dt() # self.data.set_traces_dt()
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/rwthfs/rz/cluster/home/md334167/elzerman-with-unet/HDF5Data.py", line 396, in set_traces_dt
    self.traces_dt = self.file['Traces']['Alazar Slytherin - Ch1 - Data_t0dt'][0][1]
                     ~~~~~~~~~^^^^^^^^^^
TypeError: 'NoneType' object is not subscriptable
